### ISSUE TITLE
Issue for multi-day events with iCalendar widget

### ISSUE THREAD

**Issue by awi-linkgroup** [jweiland-net/events2#168]
Hi,

I have a list with all events within a specific category.
And I changed the template to offer a ICS-Link, by entering the iCalendar-Widget.

Works fine with event that only last 1 Day.
When I have multi-day events in this list it works properly in my default language (german).
When switching to englisch, the whole page crashes:

(1/1) Error
Call to a member function getEvent() on boolean

Seems to be a localization issue.

Do you have a bugfix for this?

Regards
Adrian

**Comment by froemken** [comment]
events2 is currently not fully translatable. On my local machine I have just created all possible TYPO3 columns like deleted, sys_language_uid, ... to day table and added the related TCA configuration. For now I don't get any error while storing translated records in TYPO3 Backend. Further I have copied the iCal Widget to a list view for each event record. The export looks good  for german and english currently, but now over 10 UnitTests are broken.
